job_package_1/models/contract.py

```diff
@@ -1,10 +1,62 @@
 # -*- coding: utf-8 -*-
 
 CONTRACT_BODY = """
-<h3 style="text-align: left; ">
-	Hello
-</h3>
-<p>
-	How are you
-</p>
+<div class="points">
+    <style>							
+        .ol-strong { font-weight: bold; }
+        .ol-normal { font-weight: normal;}
+        ol {
+            list-style-type: none;
+            counter-reset: item;
+            margin: 0;
+            padding: 0;
+        }
+        ol > li {
+            display: table;
+            counter-increment: item;
+            margin-bottom: 0.6em;
+        }
+        ol > li:before {
+            content: counters(item, ".") ". ";
+            display: table-cell;
+            padding-right: 0.6em;    
+        }
+        li ol > li {
+            margin: 0;
+        }
+        li ol > li:before {
+            content: counters(item, ".") " ";
+        }
+    </style>
```

Comment:
Please delete this code for me, since we are using now external css, just so that I don't have to change anything after merging your branch